zanata-war/src/main/java/org/zanata/adapter/OpenOfficeAdapter.java

```diff
@@ -31,7 +31,7 @@
  */
 public class OpenOfficeAdapter extends OkapiFilterAdapter {
     public OpenOfficeAdapter() {
-        super(prepareFilter(), IdSource.subDocNameAndTextUnitId, true);
```

Comment:
To avoid this problem happening again, I think we should put a warning on `IdSource.subDocNameAndTextUnitId` and `IdSource.textUnitId`, perhaps deprecating if not removing them.